### PR TITLE
Docs: updating UI language guide.

### PR DESCRIPTION
**Pull request by godai78**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/contributing/git-commit-message-convention.html))

Docs: updating UI language guide.

---

### Additional information

_For example – encountered issues, assumptions you had to make, other affected tickets, etc._

### CODE REVIEW

File: docs/getting-started/setup/ui-language.md [hunk 1]
```diff
@@ -52,21 +52,27 @@ We are doing our best to deliver the best RTL support to our users and we consta
...
 	],
 	toolbar: {
-		items: [ 'undo', 'redo' ]
+		items: [ 'undo', 'redo', 'tableOfContents' ]
 	},
-	translations
+	translations: [
+		coreTranslations,
+		commercialTranslations
+		]
```
Comment (filipsobol) [inline docs/getting-started/setup/ui-language.md:75]:
```suggestion
	translations: [
		coreTranslations,
		commercialTranslations
	]
```

File: docs/getting-started/setup/ui-language.md [hunk 2]
```diff
@@ -84,23 +90,29 @@ To use different language than the default one (English), you need to load the e
 {
 	"imports": {
 		"ckeditor5": "<CDN_LINK>/ckeditor5/dist/index.min.js",
-		"ckeditor5/": "<CDN_LINK>/ckeditor5/",
+		"ckeditor5/": "<CDN_LINK>/ckeditor5/dist",
```
Comment (filipsobol) [inline docs/getting-started/setup/ui-language.md]:
```suggestion
		"ckeditor5/": "<CDN_LINK>/ckeditor5/dist",
		"ckeditor5-premium-features": "<CDN_LINK>/dist/browser/index.js",
		"ckeditor5-premium-features/": "<CDN_LINK>/dist/"
```

File: docs/getting-started/setup/ui-language.md [hunk 3]
```diff
@@ -83,24 +89,32 @@ To use different language than the default one (English), you need to load the e
 <script type="importmap">
 {
 	"imports": {
-		"ckeditor5": "<CDN_LINK>/ckeditor5/dist/index.min.js",
-		"ckeditor5/": "<CDN_LINK>/ckeditor5/",
+		"ckeditor5": "<CDN_LINK>/browser/index.js",
+		"ckeditor5/": "<CDN_LINK>/dist/",
+		"ckeditor5-premium-features": "<CDN_LINK>/dist/browser/index.js",
+		"ckeditor5-premium-features/": "<CDN_LINK>/dist/"
```
Comment (filipsobol) [inline docs/getting-started/setup/ui-language.md:95]:
```suggestion
		"ckeditor5-premium-features": "<CDN_LINK>/ckeditor5-premium-features/dist/browser/index.js",
		"ckeditor5-premium-features/": "<CDN_LINK>/ckeditor5-premium-features/dist/"
```

File: docs/getting-started/setup/ui-language.md (hunk 2 above)
Comment (godai78) [reply docs/getting-started/setup/ui-language.md]: In commit.

**Pull request merged**